src/tools/imgtool/main.cpp

```diff
@@ -760,10 +767,11 @@ static void listoptions(const util::option_guide &opt_guide, const char *opt_spe
 			break;
 		}
 
-		fprintf(stdout, "%16s %-30s %s\n",
-			opt_name.c_str(),
-			description_buffer.str().c_str(),
-			opt_desc);
+		std::wcout << wstring_from_utf8(util::string_format(
+			"%16s %-30s %s",
+			opt_name,
+			description_buffer.str(),
+			opt_desc)) << std::endl;
```

Comment:
Very interesting things are going to happen to alignment when characters are not bytes.  This may require a rethink of `string_format` to allow it to work in characters, and handle narrow/wide properly, but that's something for another time.